src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {` `}
+      {person.name}
+    </h2>
+    {person.age !== undefined && (
```

Comment:
```suggestion
    {person.age && (
```
redundant to compare with undefined. if person.age will be truthy, && operator returns second expression.